### PR TITLE
Link "conventions" language to the app model topic

### PR DESCRIPTION
**Pull request by guardrex**
@Syscall asked about the use of the word "conventions" in the topic. This PR links it over to :point_right: https://docs.microsoft.com/aspnet/core/mvc/controllers/application-model#conventions

@pranavkm Is it safe to do this, since the *Conventions* section might not take into account differences between MVC and RP?

This also:
* Changes the sample app link to our standard format.
* Surfaces where the conventions are established on the first example shown with a link to the sample app.

### CODE REVIEW

**Comment by pranavkm** [comment]
That's fine. It's actually an unfortunate design flaw that conventions don't work on both controllers and Razor Pages (like filters)

**Comment by guardrex** [comment]
@pranavkm Thanks. Do u want to approve it in its current form, or should I make changes to include something about conventions (as filters) not working on pages? I was really just concerned about the definition of "conventions" here, since it was a reader complaint about the topic.

@scottaddie How's it look?

File: aspnetcore/mvc/razor-pages/razor-pages-convention-features.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ The sample app adds a `{globalTemplate?}` route template to all of the pages in
 > [!NOTE]
 > The `Order` property for the `AttributeRouteModel` is set to `0` (zero). This ensures that this template is given priority for the first route data value position when a single route value is provided. For example, the sample adds an `{aboutTemplate?}` route template later in the topic. The `{aboutTemplate?}` template is given an `Order` of `1`. When the About page is requested at `/About/RouteDataValue`, "RouteDataValue" is loaded into `RouteData.Values["globalTemplate"]` (`Order = 0`) and not `RouteData.Values["aboutTemplate"]` (`Order = 1`) due to setting the `Order` property.
 
-*Startup.cs*:
+Razor Pages options, such as adding [Conventions](/dotnet/api/microsoft.aspnetcore.mvc.razorpages.razorpagesoptions.conventions), are added when MVC is added to the service collection in `Startup.ConfigureServices`. For an example, see the [sample app](https://github.com/aspnet/Docs/tree/master/aspnetcore/mvc/razor-pages/razor-pages-convention-features/sample/).
```
Comment (pranavkm) [inline aspnetcore/mvc/razor-pages/razor-pages-convention-features.md]: For 2.1.0-preview2, we also let you annotate types with attributes that implement a convention `class MyConventionAttribute : Attribute, IPagePropertyModelConvention`, but for `dev` this is correct

Comment (scottaddie) [reply aspnetcore/mvc/razor-pages/razor-pages-convention-features.md]: @guardrex If you haven't already, please create an issue to track this change ☝️ in 2.1.0-preview2.

**Pull request merged**